qgis2to3/codeporting/qgis_fixes/fix_pyqt.py

```diff
@@ -516,7 +517,7 @@ def transform_member(self, node, results):
                                 mod_dict.setdefault(change[0], []).append(member)
                             found = True
                     if not found:
-                        f = open("/tmp/missing", "a+")
+                        f = open(f"{tempfile.gettempdir()}/missing", "a+")
```

Comment:
The `setup.py` states: `python_requires='~=3.3'`, but you're using an f-string here which requires >= Python 3.6.  

I suspect the way forward here would be to update `setup.py` to require Python 3.6 (since that's the version shipping with QGIS 3 anyway... you'd have to be doing something strange to run these scripts with an older version of Python 3!).

The PR should perhaps be updated to either update the minimum python version, or remove the f-string to be consistent.